content/source/docs/cloud/guides/recommended-practices/part3.2.html.md

```diff
@@ -47,7 +47,7 @@ The diagram below can help you decide when to write a module:
 Spread Terraform skills to additional teams, and improve the skills of existing infrastructure teams. In addition to internal training and self-directed learning, you might want to consider:
 
 * Sign your teams up for [official HashiCorp Training](https://www.hashicorp.com/training/) .
-* Make available resources such as [Terraform Up and Running: Writing Infrastructure as Code](https://www.amazon.com/Terraform-Running-Writing-Infrastructure-Code-ebook/dp/B06XKHGJHP/ref=sr_1_1?ie=UTF8&qid=1496138592&sr=8-1&keywords=terraform+up+and+running) or [Getting Started with Terraform](https://www.amazon.com/Getting-Started-Terraform-Kirill-Shirinkin/dp/1786465108/ref=sr_1_1?ie=UTF8&qid=1496138892&sr=8-1&keywords=Getting+Started+with+Terraform). These are especially valuable when nobody in your organization has used Terraform before.
+* Make available resources such as [Terraform Up and Running: Writing Infrastructure as Code](https://www.amazon.com/Terraform-Running-Writing-Infrastructure-Code/dp/1492046906/ref=sr_1_1?keywords=terraform+up+and+running&qid=1571263416&sr=8-1) or [Getting Started with Terraform](https://www.amazon.com/Getting-Started-Terraform-Kirill-Shirinkin/dp/1786465108/ref=sr_1_1?ie=UTF8&qid=1496138892&sr=8-1&keywords=Getting+Started+with+Terraform). These are especially valuable when nobody in your organization has used Terraform before.
```

Comment:
Lists should use `-` not `*` as specified in the style guide.

https://github.com/hashicorp/engineering-docs/blob/master/writing/markdown.md#lists

Reply:
Yeah, but I tend not to overhaul the whole page's markup when doing a simple link fix; it can muddy the issue on older pages with a lot of style issues. I suppose I can update this one, though.